src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -654,11 +654,12 @@ public async Task<ActionResult> GetCommentOptions(Guid requestId)
                 r.AlwaysAccessWhen().FullControl().FullControlInternal();
                 r.AnyOf(or =>
                 {
-                    or.BeResourceOwner(new DepartmentPath(requiredDepartment).Parent(), includeParents: true, includeDescendants: true);
+                    if (!request.IsCompleted)
```

Comment:
It kind of makes sense to hide it using 403, but i just think that down the line, this logic could be subject for confusion? Will we remember in 5 month why we are doing this check?

Maybe this should be handled separatly, instead of everything in one place, just to save lines?
IMO, i would either handle this before the normal auth check or after. 

```
if (request.IsComplete) {
    return new ObjectResult(new { error = new { 
              code = "CommentsUnavailable", 
              message = "Comments are not available on completed requests"
     } }) { StatusCode = 403 }
}
```
Maybe align with the default error model (rfc prob det.)... 
This would give info on why it's 403, not just that you don't have access. I often find myself wondring why the api respond how it does, when i expect otherwise, and often it is because i have just bundled some specific logic with other, just because it saved a few lines of code :)